.tekton/coverity-availability-check-v02-push.yaml

```diff
@@ -8,8 +8,7 @@ metadata:
     pipelinesascode.tekton.dev/cancel-in-progress: "false"
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "push" && target_branch
-      == "main" && ( "task/coverity-availability-check/0.2/***".pathChanged() || ".tekton/coverity-availability-check-v02-pull-request.yaml".pathChanged()
-      )
+      == "main" && "/task/coverity-availability-check/0.2/***".pathChanged()
```

Comment:
It's not clear to me why the `.tekton/*` paths were removed from these cel expressions, don't we want the pipelines to run on any change to their yaml definition?